Parsers/ASimAuditEvent/Tests/InfobloxBloxOne_ASimAuditEvent_ASimSchemaTester.csv

```diff
@@ -0,0 +1,96 @@
+﻿Result
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [EventUid]"
+"(1) Warning: Missing recommended field [NewValue]"
+"(1) Warning: Missing recommended field [ObjectId]"
+"(1) Warning: Missing recommended field [Object]"
```

Comment:
this is actually a mandatory field in Audit schema.

![image](https://github.com/user-attachments/assets/17245d00-99c1-496b-8124-bd0fc0d218ca)

Reply:
event ObjectType is mandatory

Reply:
Done